### PR TITLE
v0.5.1

### PR DESCRIPTION
**Pull request by martindurant**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by martindurant** [comment]
@conda-forge-admin please rerender

**Comment by martindurant** [comment]
@conda-forge/core , got the above rerender failure.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,13 +14,15 @@ source:
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
   noarch: python
 
 requirements:
   build:
```
Comment (CJ-Wright) [inline recipe/meta.yaml]: I think you can get rid of the build section and add python to the host section.

**Comment by martindurant** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (martindurant) [reply recipe/meta.yaml]: The automatic #7 seems to have worked, so we could merge that and go from there

**Pull request closed without merging**